config/locales/en.yml

```diff
@@ -212,6 +212,11 @@ en:
           attributes:
             further_conditions:
               too_long: "%{name} must be %{limit} characters or fewer"
+        support_interface/application_forms/edit_applicant_details_form:
```

Comment:
Wondering if there's a way to dedupe this. Perhaps adding the text in `PhoneNumberValidator`? Not a blocker.